File: PHP/Doctrine/src/Models/Department.php
<?php

namespace Models;

use Doctrine\ORM\Mapping as ORM;
use Stringable;

/**
 * @ORM\Entity
 * @ORM\Table(name="departments")
 */
class Department implements Stringable
{
    /**
     * @ORM\Id
     * @ORM\Column(type="integer")
     * @ORM\GeneratedValue
     */
    private $id;

    /**
     * @ORM\Column(type="string")
     */
    private $name;

    public function __construct(string $name)
    {
        $this->name = $name;
    }

    public function __toString(): string
    {
        return "Department { Id: $this->id, Name: $this->name }";
    }
}

File: Kotlin/Hibernate/src/main/kotlin/Models/Department.kt
package Models

import jakarta.persistence.*

@Entity
@Table(name = "departments")
open class Department(
    @Column(nullable = false)
    open var name: String? = null
) {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    open var id: Long? = null

    override fun toString(): String = "{ Department: { Id: ${id}, Name: $name } }"
}
